kortex_driver/README.md

```diff
@@ -0,0 +1,13 @@
+# ROS 2 Kortex Driver
```

Comment:
It looks like this came from upstream commits not in our branch yet; same to the changes to hardware_interface. We may want to first do a normal merge from the upstream main branch into the orin-build branch.

Reply:
Do you want me to create a PR to merge main into the `orin-build` branch first or just remove the commit from this PR?

Reply:
Here is the [PR](https://github.com/locusrobotics/ros2_kortex/pull/4) to update `orin-build`